Author: Adam Conrad <adconrad@ubuntu.com>
Description: Disable network tests.
 disable the tests in the testsuite that require access outside our
 buildd network
Bug-Debian: http://bugs.debian.org/674374
Last-Update: 2011-12-05
Index: librest-0.7.92/tests/Makefile.am
===================================================================
--- librest-0.7.92.orig/tests/Makefile.am
+++ librest-0.7.92/tests/Makefile.am
@@ -1,4 +1,4 @@
-TESTS = proxy proxy-continuous threaded oauth oauth-async oauth2 flickr lastfm xml custom-serialize
+TESTS = proxy proxy-continuous threaded oauth2 xml custom-serialize
 # TODO: fix this test case
 XFAIL_TESTS = xml
 
